scanpy/preprocessing/_highly_variable_genes.py

```diff
@@ -335,11 +335,11 @@ def _highly_variable_pearson_residuals(
         na_position='last',
         inplace=True,
     )
-    df['highly_variable'] = False
-    df.highly_variable.iloc[:n_top_genes] = True
-    # TODO: following line raises a pandas warning
-    # (also for flavor = seurat and cellranger..)
-    df = df.loc[adata.var_names]
+
+    high_var = np.zeros(df.shape[0])
```

Comment:
I changed the initialization in order to remove the error. Please check if ok.

Reply:
thanks, that looks good!